Dockerfile

```diff
@@ -11,20 +11,20 @@ RUN apk update && \
     libffi-dev \
     bash && \
   gem install --no-ri --no-rdoc \
-    droneio \
-    --version '~> 1.0' && \
+    gli \
+    --version '~> 2.14' && \
   gem install --no-ri --no-rdoc \
     mixlib-shellout \
     --version '~> 2.2' && \
   gem install --no-ri --no-rdoc \
+    json \
+    --version '~> 2.0' && \
+  gem install --no-ri --no-rdoc \
     chef \
-    --version '~> 12.7' && \
+    --version '12.10.24' && \
```

Comment:
They move supermarket code into Chef core, but it was buggy (several open issues and 1 I hit).  So I pinned Chef for now and am continuing to use the knife-supermarket gem for now.